sklearn/manifold/t_sne.py

```diff
@@ -142,10 +142,12 @@ def _kl_divergence(params, P, degrees_of_freedom, n_samples, n_components,
 
     # Q is a heavy-tailed distribution: Student's t-distribution
     n = pdist(X_embedded, "sqeuclidean")
-    n += 1.
```

Comment:
Hi Alex, sure! 

My result certainly agrees with the equation in that paper for the important part - which is that the implementation of q_ij needs to divide alpha *before* adding 1, instead of after (as is the present sklearn behaviour) -- this made it look like it was working because of course for 1 degree of freedom (which is most often used) these are equal.